remo/settings.py

```diff
@@ -167,6 +167,9 @@
         'visibility_timeout': config('REDIS_VISIBILITY_TIMEOUT', default=604800, cast=int)
     }
 
+# Product Details
+PROD_DETAILS_DIR = Path(ROOT, 'lib/product_details_json')
```

Comment:
Why is this line needed?

Reply:
This is for downloading product-details files into the lib folder. Running update products command will fetch new files.